### PR TITLE
Keep date visible on scrollytelling maps

### PR DESCRIPTION
**Pull request by danielfdsilva**
Ensures that the date remains visible by changing the scrollytelling block offset when the header is visible #376

### CODE REVIEW

**Review by aboydnw — APPROVED** [review]
Functionality-wise, looks good to me 👍

File: app/scripts/components/common/blocks/scrollytelling/index.tsx [hunk 1]
```diff
@@ -48,14 +49,15 @@ export const scrollyMapHeight = 'calc(100vh - 3rem)';
...
-  top: 3rem;
+  transition: top ${HERO_TRANSITION_DURATION}ms ease-out,
+    height ${HERO_TRANSITION_DURATION}ms ease-out;
 
-  ${media.mediumUp`
-    height: calc(100vh - 4rem);
-    top: 4rem;
+  ${({ topOffset }) => css`
+    top: ${topOffset}px;
+    height: calc(100vh - ${topOffset}px);
```
Comment (nerik) [inline app/scripts/components/common/blocks/scrollytelling/index.tsx:60]: Is that the equivalent of the following?:
```
top: ${({ topOffset }) => topOffset}px;
height: calc(100vh - ${({ topOffset }) => topOffset}px);
```

Not saying it's any better, but the [styled components doc](https://styled-components.com/docs/api#css) makes it confusing to me

**Review by nerik — APPROVED** [review]
LGTM!

File: app/scripts/components/common/blocks/scrollytelling/index.tsx [hunk 2]
```diff
@@ -37,6 +36,8 @@ import { MapLoading } from '$components/common/loading-skeleton';
 import { HintedError } from '$utils/hinted-error';
 import { formatSingleDate } from '$components/common/mapbox/utils';
 import { convertProjectionToMapbox } from '$components/common/mapbox/projection-selector/utils';
+import { useSlidingStickyHeaderProps } from '$components/common/layout-root';
+import { HERO_TRANSITION_DURATION } from '$components/analysis/page-hero-analysis';
```
[the remark this answers is not in the document]
Comment (danielfdsilva) [reply app/scripts/components/common/blocks/scrollytelling/index.tsx:40]: Fair point

[the remark this answers is not in the document]
Comment (danielfdsilva) [reply app/scripts/components/common/blocks/scrollytelling/index.tsx]: @nerik Addressed :)

**Pull request merged**